backend/app/services/user_notifications.rb

```diff
@@ -37,7 +37,9 @@ def deliver_new_studies
       return unless users.any?
 
       users.each_value do |user|
-        UserMailer.with(user:, studies:).new_studies.deliver
+        studies.each do |study|
```

Comment:
with looping over each study, would we be emailing user A for every new study? ie would they get 5 emails if there were 5 new studies? is that what we want?